tests/run-macros/tasty-custom-show/quoted_1.scala

```diff
@@ -38,9 +38,10 @@ object Macros {
     '{print(${buff.result()})}
   }
 
-  def dummyShow(given qctx: QuoteContext): qctx.tasty.Printer = {
-    import qctx.tasty._
-    new Printer {
+  def dummyShow(given qctx: QuoteContext): scala.tasty.reflect.Printers[qctx.tasty.type]#Printer = {
```

Comment:
The type signature does not look nice.

Reply:
I agree. Would be improved if I manage to decouple the contents of `Printers`